tests/terraform/TestSetMetadataTags/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_instance" "no_protection" {
```

Comment:
this only needs the one instance for metadata tags

Reply:
removed, Thanks